README.md

```diff
@@ -62,41 +66,50 @@ I write my personal thoughts on :gear: Tech & :seedling: Life in my :writing_han
 * I feel more focused with a hoodie puts on
 * I wake up at 4:40 followed with a cold shower
 * I'm a huge fan of Iron Man and Hulk
-* I'm a big fan of *Universe of Metro 2033* - mostly books and books "Daemon" and "Freedom" written by Daniel Suarez
+* I really enjoyed *Universe of Metro 2033* books series; also a big fan of two books: "Daemon" and "Freedom" written by Daniel Suarez.
 * I use ![Todoist](https://img.shields.io/badge/-Todoist-E44332?logo=Todoist&logoColor=fff) everyday
 
 ## :hourglass_flowing_sand: Things I don't know, I want to master or just improve (randomly ordered)
 
-* Software Architecture, microservices and distributed systems as a whole
+* Software Architecture, microservices, distributed systems and all the fancy keywords
+* Dive into Domain-driven design
+* Software Design Patterns
 * Software Craftsmanship
 * Learn teaching and mentoring
 * code search and read with **grep** and **find** from command line (only ack-grep right now)
-* best practices API design and security
+* best practices related with API design and its security
 * Golang
 * Vuetify & TailwindCSS
-* Docker ecosystem & Kubernetes
-* Building and managing CI/CD: Travis CI & GitHub Actions
+* deeper understanding of Docker ecosystem with Kubernetes
+* Building and managing CI/CD pipes: Travis CI & GitHub Actions.
 * Webpack
-* Mermaid and maybe PlantUML
+* Mermaid and **maybe** PlantUML
 * BPML
 * AWS (mostly web applications related services)
-* GraphQL
+* GraphQL, Elasticsearch, Redis, MongoDB, Firebase
 * Responsive Web Design & Flexbox
 * Progressive Web Application
 * Web Applications Security
 * **maybe** ReactJS and React Native
-* Flutter & Kotlin
+* **maybe** Flutter & Kotlin
 * Electron
 * TypeScript
 * Messaging queues (RabbitMQ, ZeroMQ, RQ)
 * Asynchronous programming (asyncio, threading, Promises)
 * Reactive programming
-* Testing: E2E, TDD, BDD
+* Quality Assurance: E2E, TDD, BDD: pytest, jest, Selenium, Cypress, Behave, Lettuce, Robot Framework
 * Arduino & AVR programming, scratch the surface with Raspberry Pi
-* Blender, Inkscape, Aseprite - mostly for pixelart and gamedev :stars:
-* Elasticsearch, Redis, MongoDB, Firebase
-* Online Marketing (Google, Facebook, Amazon, Pinterst) & SEO
-* Network troubleshooting (CCNA or something)
+* Inkscape, Aseprite, Blender - mostly for pixelart and gamedev :stars:
+* Online Marketing (focusing on Google, Facebook, Amazon & Pinterst) & SEO
+* Network troubleshooting (CCNA or something); building and managing home networks
+* Web Scraping (with Scrapy package)
 * ...
 
+## :books: Currently Reading
+
+* *48-godzinna doba* - Maciej Wieczorek
```

Comment:
it would be great to post some summary after finishing reading those books. Eg. on your blog

Reply:
Good idea, I'll think about it.